modules/subchannel/examples/MultiApp/fuel_assembly.i

```diff
@@ -264,51 +252,36 @@ duct_inside = '${fparse duct_outside - 2 * duct_thickness}'
   []
 []
 
-[UserObjects]
-  active = ''
-  [Tpin_avg_uo]
-    type = NearestPointLayeredAverage
-    direction = z
-    num_layers = 1000
-    variable = Tpin
-    block = fuel_pins
-    points = '${fparse 0.012} 0.0 0.0'
-    execute_on = timestep_end
-  []
-[]
-
 [Outputs]
   exodus = true
+  csv = true
 []
 
 [Executioner]
   type = Transient
-  petsc_options_iname = '-pc_type -pc_hypre_type'
-  petsc_options_value = 'hypre boomeramg'
```

Comment:
are these unused in subchannel? Or are we now defaulting to ILU?

Reply:
unused